### ISSUE TITLE
Favorites list drop down

### ISSUE THREAD

**Issue by chintan97** [sumatrapdfreader/sumatrapdf#977]
For particular file, if we create more than 10 favorites; it only shows first 10 favorites(image 1). For sample, I have created 12 favorites in my file(image 2). To overcome issue, add up-down arrows to scroll in list; sample is attached(image 3).

Image 1:
![bug_2](https://user-images.githubusercontent.com/16590946/36540935-3f174718-1802-11e8-9b67-e962add22537.png)

Image 2:
![bug_1](https://user-images.githubusercontent.com/16590946/36540943-46ec055a-1802-11e8-9830-55ccc9dad1a9.png)

Image 3:
![bug_3](https://user-images.githubusercontent.com/16590946/36540889-19be56be-1802-11e8-8df3-d9408aa47596.png)

**Comment by GitHubRulesOK** [comment]
Hmm @kjk not sure that this was what the OP meant exactly but it seems to be an alternative when used in context of add a shortcut or a text icon !

<img width="763" height="447" alt="Image" src="https://github.com/user-attachments/assets/56099413-6620-4d90-95ea-b7d5d7318f06" />

**Comment by kjk** [comment]
Agreed, `Ctrl + k`, `$` is recommended way of quickly navigating favorites.
